PopulateTools/gobierto
# Fix publishers and subscribers constants autoload

Bugfix

### What does this PR do?

This PR fixes const autoload for PubSub modules in development mode. It was an issue with the const missing and the names of the constants:

1. I have removed the PubSub namespace
2. I have added a couple of require dependencies in the `config/application.rb`

More information: [Autoloading and Reloading Constants](http://guides.rubyonrails.org/autoloading_and_reloading_constants.html#require-dependency)

### How should this be manually tested?

It was very easy to test, because it happens when an event is emitted and the code of the application has changed:

1. Visit some CMS page edit and update it in your browser
2. Edit some code from rails, for example, a method in the Site model
3. Try to update the CMS page again
4. it should work now
